core/src/main/resources/hudson/model/Run/confirmDelete.jelly

```diff
@@ -36,8 +36,8 @@ THE SOFTWARE.
 
       <j:if test="${msg==null}">
         <form method="post" action="doDelete">
-          ${%delete.build(it.displayName)}
-          <f:submit value="${%Yes}" clazz="jenkins-!-destructive-color" />
+          <span style="margin-right: 1em">${%delete.build(it.displayName)}</span>
```

Comment:
We could always use one of the spacing classes from https://weekly.ci.jenkins.io/design-library/Spacing/ however it might be worth just having the button on a new line.

Reply:
@davidho0403 Could you update your PR please, to use one of the spacings matching, like outlined above, please?